plugins/rikki/heroeslounge/classes/heroes/HeroUpdater.php

```diff
@@ -111,6 +116,11 @@ public static function setTalentImage($talent_model, $icon_url)
             mkdir($talent_image_path, 0777, true);
         }
 
+        // Check if the file already exists and if it's larger than 500 bytes.
```

Comment:
nit: This comment does not add anything that isnt obvious in the line below, so just delete it.